operator/v1/types.go

```diff
@@ -45,6 +45,23 @@ type ClusterManagerSpec struct {
 	// NodePlacement enables explicit control over the scheduling of the deployed pods.
 	// +optional
 	NodePlacement NodePlacement `json:"nodePlacement,omitempty"`
+
+	// DeployOption contains the options of deploying a cluster-manager
+	// +optional
+	DeployOption DeployOption `json:"deployOption,omitempty"`
+}
+
+type DeployOption struct {
+	// Mode can be Default or Hosted.
+	// In Hosted mode, hub hosting all deployments on another cluster.
```

Comment:
Which ns the deployments are running on with hosted mode?

The following sentence should also be changed
```
// ClusterManager will only be deployed in open-cluster-management-hub namespace.
```

Reply:
In hosted mode, only crd and configurations are installed on the hub. Controllers are run in another cluster (defined as hosted cluster) and connect to the hub with the kubeconfig in secret of  "external-hub-kubeconfig".